<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="colorSelectWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>362</width>
      <height>234</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/Images/ColorPalette.jpg);
border: 1px solid rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QWidget" name="colorShowWidget" native="true">
    <property name="geometry">
     <rect>
      <x>422</x>
      <y>30</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
background-color: white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hexInput">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>92</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>422</x>
      <y>140</y>
      <width>65</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HEX: #</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>299</y>
      <width>591</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>422</x>
      <y>195</y>
      <width>160</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>RGB ↔ LAB ↔ CMYK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB ↔ CMYK ↔ HLS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB ↔ XYZ ↔ LAB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB ↔ HSV ↔ LAB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CMYK ↔ LAB ↔ HSV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CMYK ↔ RGB ↔ HLS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CMYK ↔ RGB ↔ HSV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB ↔ XYZ ↔ HSV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HSV ↔ XYZ ↔ LAB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CMYK ↔ LAB ↔ RGB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XYZ ↔ LAB ↔ HLS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB ↔ XYZ ↔ HLS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB ↔ XYZ ↔ CMYK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CMYK ↔ LAB ↔ XYZ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB ↔ CMYK ↔ HSV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CMYK ↔ HLS ↔ XYZ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB ↔ HLS ↔ LAB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CMYK ↔ XYZ ↔ RGB</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
